app/src/main/res/layout/fragment_notification.xml

```diff
@@ -1,18 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".NotificationFragment"
-    android:background="#121212">
+    android:background="#121212"
```

Comment:
Question: will this value be used again somewhere else?

Reply:
i dont remember deleted this? hmmmm